src/main/kotlin/it/pagopa/ecommerce/cdc/exceptions/LockException.kt

```diff
@@ -0,0 +1,4 @@
+package it.pagopa.ecommerce.cdc.exceptions
+
+sealed class LockException(message: String, throwable: Throwable? = null) :
```

Comment:
another non-blocking suggestion
```suggestion
sealed class EventLockException(message: String, throwable: Throwable? = null) :
```